### ISSUE TITLE
[Package Issue]: Fortinet.FortiClientVPN very outdated

### ISSUE THREAD

**Issue by itsTyrion** [microsoft/winget-pkgs#50157]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Other

### Brief description of your issue

Winget has FortiClient VPN version 6.2.0.0780 (August 2019). The latest version at the time of writing is 7.0.2 (October 2021).

### Steps to reproduce

Doesn't apply

### Actual behavior

Doesn't apply

### Expected behavior

Doesn't apply

### Environment

```shell
Doesn't apply
```

**Comment by Trenly** [comment]
@itsTyrion Can you provide the download links to the new version’s installer?

**Comment by vedantmgoyal2009** [comment]
https://links.fortinet.com/forticlient/win/vpnagent

**Comment by Trenly** [comment]
> https://links.fortinet.com/forticlient/win/vpnagent

Looks like they moved to an interactive only installer; I can't seem to figure out the TOS acceptance switch

**Comment by itsTyrion** [comment]
> Looks like they moved to an interactive only installer; I can't seem to figure out the TOS acceptance switch

Not quite, they moved to "online installer downloads offline installer and executes that". 

%LocalAppData%\Temp\FortiClientVPN.exe is the full path of what's downloaded.
If you run THAT with `.\FortiClientVPN.exe /quiet /norestart`, you can do a non-interactive install

**Comment by SpecterShell** [comment]
It is impossible to get the URL of the offline installer from the online installer because it fetches the files from encrypted API. 😕
![image](https://user-images.githubusercontent.com/56779163/156867247-29f37dd8-b1ff-48db-9661-cb5ffe6354cd.png)

**Comment by itsTyrion** [comment]
Whhhyyyy
